### PR TITLE
COMMERCE-4616 Implement the correct colors in the order and shipment progress bars

### PR DESCRIPTION
**Pull request by riccardo-alberti**

### CODE REVIEW

**Comment by riccardo-alberti** [comment]
ci:test:sf

**Comment by riccardo-alberti** [comment]
ci:test:relevant

**Comment by riccardo-alberti** [comment]
ci:test:sf

**Comment by riccardo-alberti** [comment]
ci:test:relevant

**Comment by marco-leo** [comment]
ci:test:relevant

File: modules/apps/commerce/commerce-frontend-js/src/main/resources/META-INF/resources/styles/_variables.scss [hunk 1]
```diff
@@ -32,9 +32,13 @@ $background-color: #f1f2f4;
 $border-color: #e7e7ed;
 $info-bc: #eef2fa;
 
-$step-tracker-completed: #6fcf97;
-$step-tracker-active: #6b6c7e;
-$step-tracker-inactive: #cdced9;
+$step-tracker-completed: #3ccd95;
```
Comment (marco-leo) [inline modules/apps/commerce/commerce-frontend-js/src/main/resources/META-INF/resources/styles/_variables.scss:35]: @riccardo-alberti can you please sort

**Review by marco-leo — CHANGES_REQUESTED** [review]

**Comment by riccardo-alberti** [comment]
ci:test:sf

**Comment by marco-leo** [comment]
ci:test:relevant

**Review by dsitu — APPROVED** [review]

**Comment by marco-leo** [comment]
ci:test:relevant

**Comment by dsitu** [comment]
ci:forward

**Pull request closed without merging**